x/reward/module.go

```diff
@@ -149,12 +148,18 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 func (AppModule) ConsensusVersion() uint64 { return 2 }
 
 // BeginBlock returns the begin blocker for the reward module.
-func (am AppModule) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) {
-	BeginBlocker(ctx, req, am.keeper, am.bankKeeper, am.stakingKeeper, am.distKeeper)
+func (am AppModule) BeginBlock(ctx context.Context) error {
+	return BeginBlocker(ctx, am.keeper, am.bankKeeper, am.stakingKeeper, am.distKeeper)
 }
 
 // EndBlock returns the end blocker for the reward module. It returns no validator
 // updates.
-func (AppModule) EndBlock(_ sdk.Context, _ abci.RequestEndBlock) []abci.ValidatorUpdate {
-	return []abci.ValidatorUpdate{}
+func (AppModule) EndBlock(_ context.Context) error {
```

Comment:
We can remove this func.
And we can also remove `RegisterInvariants` func

Reply:
https://github.com/xpladev/xpla/pull/154/commits/4ba12fe987d77212ca050c7abb0c66b2c4714da7